shim.c

```diff
@@ -1482,24 +1541,28 @@ load_certs(EFI_HANDLE image_handle)
 		goto done;
 	}
 
```

Comment:
The set of diffs below are not as strange as they look: load_sbat_level_file(image_handle, PathName); is added and then everything just moved into if (secure_mode()) { }
The diff just hung onto the empty line because it's not smart enough to ignore indenting.